rules-reviewed/openshift/logging.windup.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0"?>
+<ruleset id="logging"
+    xmlns="http://windup.jboss.org/schema/jboss-ruleset"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This is a ruleset for logging-related topics when migrating and application to OpenShift environments.
+        </description>
+        <dependencies>
+          <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final"/>
+          <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="java"/>
+        <sourceTechnology id="java-ee"/>
+        <targetTechnology id="openshift"/>
+        <targetTechnology id="cloud-readiness"/>
+        <tag>logging</tag>
+    </metadata>
+    <rules>
```

Comment:
@mrizzi I think that it might be a mistake to leave out the common wrapper loggers as well. It may be that we won't detect their logging framework if they only use slf4j, but they would still need to change the configuration.